spec/services/price_policies/time_based_price_calculator_spec.rb

```diff
@@ -328,4 +354,51 @@
       end
     end
   end
+
+  context "when instrument has daily rate pricing" do
```

Comment:
Can we add a spec for edge cases? Even when it should be the same, I think it's worth it to test a reservation that starts at 00:00

Reply:
Sure